engine/src/macosx_math.h

```diff
@@ -25,15 +27,15 @@
 /// works on *any* float... INF is 0x7ffffff and -INF is -(1<<31) NaN is 0
 extern int float_to_int(float);
 extern int double_to_int(float); //works on *any* float
-#if defined (__APPLE__) || defined (MACOSX)
```

Comment:
Is this really necessary today? Why wouldn't the same code work for Linux and Mac?
I have to check (again) and until 2006, Macs came with a power pc. Surely that was the reason for the discrepancy.

Reply:
I don't know the reason for the discrepancy. Absent that knowledge, I prefer to leave the code mostly as-is.

Can we table this for now, and come back to it at a future date? Like next week, maybe?